src/main/scala/cromwell/engine/ExecutionTable.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.engine
+
+import cromwell.binding.{Call, WdlBinding}
+
+/**
+ * Corresponds to the "execution table" of our discussions.
+ */
```

Comment:
I'm okay with the `Symbols`.  `ExecutionState` feels too much like an enum and not a collection of things.  Maybe pluralize it?  I still prefer `SymbolTable` and `ExecutionTable` personally but I don't mind `Symbols` and `ExecutionStates` if you have really strong feelings about it.

I liked the 'Table' suffix originally because I just thought these data structures were 'tabular' not necessarily reflecting the way in which they're stored.  Like, if you were to display these on a web page you'd more likely than not use a `<table>` tag, that's all I meant by it.  When we draw them out on the board, we draw them as tables too and we use terms like 'row' and 'column' when talking about it.